spec.html

```diff
@@ -34532,8 +34534,205 @@ <h2>Syntax</h2>
           [+UnicodeMode] `-`
           CharacterClassEscape[?UnicodeMode]
           CharacterEscape[?UnicodeMode]
+
```

Comment:
I think a better order for the productions would be:
```
ClassContents
ClassUnion
ClassRange
ClassIntersection
ClassSubtraction
ClassOperand
NestedClass
ClassStrings
MoreClassStrings
ClassString
NonEmptyClassString
ClassCharacter
ClassReservedDouble
ClassSyntaxCharacter
ClassAllowEscaped
```

Reply:
I prefer keeping the union/intersection/subtraction set operations together. Otherwise I don't care much and moved productions as requested. PTAL: b82a372